### PR TITLE
Enable notary support

### PR DESCRIPTION
**Pull request by jsuchome**
Point to SLES based images

### CODE REVIEW

**Comment by jsuchome** [comment]
Do not merge until the images are built and published...

**Review by stefannica — DISMISSED** [review]

**Review by flaviodsr — CHANGES_REQUESTED** [review]
Functionality wise the change is good, I have tested and signing images works. However I think the name of the images are not right...

File: harbor-helm/values.yaml [hunk 1]
```diff
@@ -605,13 +605,13 @@ trivy:
...
 notary:
-  enabled: false
+  enabled: true
   server:
     # set the service account to be used, default if left empty
     serviceAccountName: ""
     image:
-      repository: goharbor/notary-server-photon
-      tag: v2.0.2
+      repository: registry.suse.com/registry/notary-server
```
[the remark this answers is not in the document]
Comment (jsuchome) [reply harbor-helm/values.yaml:613]: good point

**Review by flaviodsr — APPROVED** [review]

**Comment by stefannica** [comment]
Closing in favor of #58

**Pull request closed without merging**